homeassistant/components/vera/common.py

```diff
@@ -27,3 +29,37 @@ def get_configured_platforms(controller_data: ControllerData) -> Set[str]:
         platforms.append(SCENE_DOMAIN)
 
     return set(platforms)
+
+
+class SubscriptionRegistry(pv.AbstractSubscriptionRegistry):
+    """Manages polling for data from vera."""
+
+    def __init__(self, hass: HomeAssistant) -> None:
+        """Initialize the object."""
+        super().__init__()
+        self._hass = hass
+        self._cancel_poll = None
+
+    def start(self) -> None:
+        """Start polling for data."""
+        self.stop()
+        self._schedule_poll(1)
+
+    def stop(self) -> None:
+        """Stop polling for data."""
+        if self._cancel_poll:
+            self._cancel_poll()
+
+    def _schedule_poll(self, delay: float) -> None:
+        self._cancel_poll = async_call_later(self._hass, delay, self._run_poll_server)
```

Comment:
This will be called by sync context after the first iteration, so we can't use the async api.
```suggestion
        self._cancel_poll = call_later(self._hass, delay, self._run_poll_server)
```